aesara/graph/fg.py

```diff
@@ -125,6 +131,13 @@ def __init__(
             )
             outputs = [memo[o] for o in outputs]
             inputs = [memo[i] for i in inputs]
+        else:
+            # If the graph starts from Ops that
+            # don't have input variables.
+            for i, _inp in enumerate(inputs):
+                if _inp.owner and len(_inp.owner.inputs) == 0:
+                    _inp.owner = None
+                    _inp.index = None
```

Comment:
Not exactly sure if this is how it's supposed to work.

An example of this case is in:
https://github.com/aesara-devs/aesara/blob/620edabea58853a581cc715e3fe407ea4e63520d/tests/graph/test_fg.py#L13-L17

Reply:
What is the "it" in this situation?

Reply:
Ah, this part of the code has been discussed and removed. This part dealt with variables with non-`None` owners since the were not allowed as inputs. Now that restriction (along with the code tagged above) has been removed and doesn't look like it will be a problem as of now.